DTAdmin/Subject/Question/Controllers/QuestionsTableViewController.swift

```diff
@@ -102,9 +109,11 @@ class QuestionsTableViewController: UITableViewController, UISearchBarDelegate {
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "questionCell", for: indexPath)
+        let cell = tableView.dequeueReusableCell(withIdentifier: "questionCell", for: indexPath) as! QuestionTableViewCell
         let cellData = inSearchMode ? filteredData[indexPath.row] : questions[indexPath.row]
-        cell.textLabel?.text = "\(indexPath.row + 1). " + cellData.questionText
+        cell.setQuestion(question: cellData)
+        cell.delegate = self
+        //cell.textLabel?.text = "\(indexPath.row + 1). " + cellData.questionText
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.